packages/babel-parser/src/parser/expression.js

```diff
@@ -1708,6 +1708,7 @@ export default class ExpressionParser extends LValParser {
       this.finishNode(val, "SequenceExpression");
       val.end = innerEndPos;
       val.loc.end = innerEndLoc;
+      if (this.options.ranges) val.range[1] = innerEndPos;
```

Comment:
We can call `resetEndLocation` on `val`.

Reply:
Thanks 6f2ae6c